python/sbp/client/loggers/json_logger.py

```diff
@@ -62,13 +67,17 @@ def next(self):
 
     """
     for line in self.handle:
-      data = json.loads(line)
-      delta = data['delta']
-      timestamp = data['timestamp']
-      item = SBP.from_json_dict(data['data'])
       try:
-        yield (delta, timestamp, self.dispatcher(item))
-      except KeyError:
-        yield (delta, timestamp, item)
+        data = json.loads(line)
+        delta = data['delta']
+        timestamp = data['timestamp']
+        item = SBP.from_json_dict(data['data'])
+        try:
+          yield (delta, timestamp, self.dispatcher(item))
+        except KeyError:
+          yield (delta, timestamp, item)
+      except ValueError:
```

Comment:
The indenting of the `ValueError` should be inside `for line in self.handle`:

```python
    for line in self.handle:
      try:
        data = json.loads(line)
        delta = data['delta']
        timestamp = data['timestamp']
        item = SBP.from_json_dict(data['data'])
        try:
          yield (delta, timestamp, self.dispatcher(item))
        except KeyError:
          yield (delta, timestamp, item)
      except ValueError:
        warn = "Bad JSON decoding for line %s" % line
        warnings.warn(warn, RuntimeWarning)
```

Should just go to the next line, right?